lib/utilities/reportHandler.js

```diff
@@ -22,6 +22,26 @@ function isAuthorized(clientIP, req) {
         req.headers['x-scal-report-token'] === config.reportToken;
 }
 
+const itemScanRefreshDelay = 1000 * 30 * 60; // 30 minutes
+let lastItemScanTime = undefined;
```

Comment:
I may be wrong but I don't believe so, `require`'d modules are cached so module-init code does not run several times.